share/mrdocs/addons/generator/asciidoc/partials/signature/alias.adoc.hbs

```diff
@@ -0,0 +1,5 @@
+{{!-- alias.adoc.hbs --}}
+alias {{symbol.name}} = {{>declarator symbol.aliasedType}}
+{{#if symbol.javadoc}}
+{{>javadoc symbol.javadoc}}
```

Comment:
The signature of a symbol doesn't include javadocs... nevermind the fact that there is no `javadoc` partial